### PR TITLE
Scenario import from another file

### PR DESCRIPTION
**Pull request by rmacor**
This PR will allow for scenario in benchmark configuration to be imported from another file. Import is possible from another benchmark file or from designated scenario file - containing only scenario. PR also creates separate XML scheme for scenario and tests for scenario parsing including tests for: scenario imported from URL, from another benchmark, from designated file and without scenario import.

### CODE REVIEW

**Comment by mcimbora** [comment]
This applies to all classes - please make sure the formatting is right (e.g. use spaces between class names and curly brackets, remove additional spaces, etc.)

**Comment by rvansa** [comment]
@mcimbora If you want to be pedantic about code formatting, you can play with checkstyle plugin https://maven.apache.org/plugins/maven-checkstyle-plugin/ - though, that would mean fixing a lot of current non-compliant code.

**Comment by mcimbora** [comment]
@rmacor That's it, thanks.

**Comment by mcimbora** [comment]
@rvansa Nice, generally I'd love to have at least a code formatting template. I'll create an issue for this.

**Comment by rmacor** [comment]
PR updated.

**Comment by mcimbora** [comment]
Thanks @rmacor, this looks good to me. Do you want to have a look at the PR  @alanfx? I'll go ahead and integrate it if there are no more comments.

**Comment by alanfx** [comment]
@mcimbora yeah, I'll look at it as well. I was looking at your PR!

**Comment by alanfx** [comment]
@mcimbora Looks good to me, @rmacor. I would like to get these classes documented as long as they are being updated, though.

**Comment by rmacor** [comment]
PR updated - some documentation added.

File: core/src/main/java/org/radargun/config/DomConfigParser.java [hunk 1]
```diff
@@ -100,6 +102,20 @@ public MasterConfig parseConfig(String config) throws Exception {
       return masterConfig;
    }
 
+   private Element loadScenario(String scenarioFile) throws ParserConfigurationException {
```
Comment (mcimbora) [inline core/src/main/java/org/radargun/config/DomConfigParser.java]: As for the documentation that @alanfx suggested, this is probably the piece of code that would deserve it most (as this is the main idea of the PR). I'd also change the name of parameter to 'scenarioUri'. The rest looks good to me. I'll integrate the changes once fixed.

**Comment by rmacor** [comment]
PR updated - documentation to loadScenario method added.

**Comment by mcimbora** [comment]
Integrated, thanks.

**Pull request closed without merging**